src/viam/examples/project/pkg-config/Makefile

```diff
@@ -1,19 +1,24 @@
 CXX = c++
 
-CXXFLAGS = -std=c++14
-CXXFLAGS += `pkg-config --cflags viam-cpp-sdk-libviamsdk`
-LINKFLAGS = `pkg-config --libs viam-cpp-sdk-libviamsdk`
+VIAM_PREFIX ?= ../../../../../build/install
+PKG_CONFIG_PATH ?= $(VIAM_PREFIX)/lib/pkgconfig
+
+
+CXXFLAGS = -std=c++17
```

Comment:
Is this a necessary change? It seems worth discussing a change in our default version before pushing this.

Reply:
Good question, and I think the answer is "sorta kinda". Abseil forces a C++ version on its consumers, so if the version is new enough and you're passing C++14 to Make then it would error out. I think for the goal of making it work out of the box we might want this, or maybe to put a comment or turf it to another `?=` variable

Reply:
I'm pretty OK with this. C++14 was a conservative choice when the SDK was started ~2 years ago. C++17 is long since mainstream, and we should probably consider making it our version of record in the near future.

Reply:
I do wonder though if it should be `+=` in case the caller has passed in some `CXXFLAGS` already on the command line / environment.

Reply:
So these are some good points/questions, and I guess here we want to differentiate between what we're requiring for the SDK itself, and what we require for this example project, keeping in mind the goal that this Makefile should be an inviting "works out of the box" example.

I think my vote would be to have a line like

```Makefile
# The SDK technically supports c++14, but if you have a relatively recent 
# install of abseil it will force c++17
CXX_STANDARD ?= c++17

# ...

CXXFLAGS += -std=$(CXX_STANDARD) ...
```